samples/run-series_2nx.sh

```diff
@@ -14,8 +15,11 @@ do
 	rm -f junk2
 	head -c $nbyte $1 > junk2;
 	ls -l junk2;
-	numactl -N 0 ./compdecomp_th junk2 $th $rpt &
-	numactl -N 8 ./compdecomp_th junk2 $th $rpt &
+	for i in $nodes
```

Comment:
Should this file define the variable `nodes` too?

Reply:
Thanks, yes. Will correct this and resend it.